TRANSLATING.md

```diff
@@ -33,12 +33,14 @@ Now, lets start looking at texts to translate.
 Click on Adventures to reach this screen:
 ![Weblate tour - Adventures](https://user-images.githubusercontent.com/28646458/156937285-2418f2b9-5dcb-46ba-85e4-ad8266a8aae4.jpg)
 
-Now you see an overview of this component. Little to much information maybe. If so, choose "Labeled: Automatically translated". These are strings that have the English text in place of the translation. This is the way we set up new texts. This way every text that has not been translated yet, is at least visible in English on the website. 
+Now you see an overview of this component. Little to much information maybe. Don't worry, just start with "Labeled: Automatically translated". These are strings that have the English text in place of the translation. This is the way we set up new texts. This way every text that has not been translated yet, is at least visible in English on the website. 
```

Comment:
to -> too

This is the way we set up new texts -> voor nieuwe mensen is dit natuurlijk niet nieuw :) Ik zou 'new' weghalen.